hebiros_advanced_examples/rviz/omnibase_gazebo_model.rviz

```diff
@@ -0,0 +1,287 @@
+Panels:
```

Comment:
I couldn't find where this file is used? It wasn't used in mapping or navigation launch files...